### PR TITLE
fix: [CDS-68849]: Added expressionRender for cloudformation tags

### PR DESCRIPTION
**Pull request by satyam-harness**
#### Screenshots
before
image

after 
image

NA

#### PR Checklist

<details>
<summary>Please check if your PR fulfils the following requirements</summary>

- [ ] Tests for the changes have been added. Ideally, include a test that fails without this PR but passes with it.
- [ ] Docs have been [added/updated](https://harness.atlassian.net/jira/software/c/projects/DOC/boards/40).
</details>

<details>
<summary>Use the following comments to re-trigger PR Checks</summary>

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Sonar: `retrigger sonar`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Feature Name Check: `trigger featurenamecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress Rest: `retrigger cypress-rest`
- Cypress CD: `retrigger cypress-cd`
- Cypress Pipeline: `retrigger cypress-pipeline`
- Cypress CV: `retrigger cypress-cv`
- Fix Prettier: `fix prettier`
</details>

### CODE REVIEW

**Review by AshwinBhatkal-Harness — APPROVED** [review]

File: src/modules/10-common/components/MonacoTextField/MonacoTextField.tsx [hunk 1]
```diff
@@ -65,7 +65,6 @@ export function MonacoText(props: ConnectedMonacoTextFieldProps): React.ReactEle
   const [isFullScreen, setFullScreen] = React.useState(false)
   const { getString } = useStrings()
   const value = get(formik.values, name) || ''
```
Comment (AshwinBhatkal-Harness) [inline src/modules/10-common/components/MonacoTextField/MonacoTextField.tsx:67]:
```suggestion
  const value = get(formik.values, name) || ''
  
```

File: src/modules/10-common/components/MonacoTextField/MonacoTextField.tsx [hunk 2]
```diff
@@ -65,7 +65,7 @@ export function MonacoText(props: ConnectedMonacoTextFieldProps): React.ReactEle
...
-
+
```
Comment (AshwinBhatkal-Harness) [inline src/modules/10-common/components/MonacoTextField/MonacoTextField.tsx:68]:
```suggestion

```

File: src/modules/10-common/components/MonacoTextField/MonacoTextField.tsx (hunk 1 above)
Comment (AshwinBhatkal-Harness) [inline src/modules/10-common/components/MonacoTextField/MonacoTextField.tsx:67]:
```suggestion
  const value = get(formik.values, name) || ''

```

**Review by AshwinBhatkal-Harness — APPROVED** [review]

**Review by Sagar-Rajput7 — APPROVED** [review]

**Comment by satyam-harness** [comment]
retrigger titlecheck

**Pull request merged**